content/api/commands/task.md

```diff
@@ -306,8 +321,9 @@ are using
 [cypress-skip-and-only-ui](https://github.com/bahmutov/cypress-skip-and-only-ui)
 plugin and want to install your own task to read a file that might not exist:
 
-```javascript
-// in plugins/index.js file
+:::cypress-plugin-example
```

Comment:
Let's confirm this is what we want here since the example changed a bit with the new directive.

Reply:
Wrapped the comments up in these fixes cecc092f4f4d04ec2bc24b0b48bbcf4968720671